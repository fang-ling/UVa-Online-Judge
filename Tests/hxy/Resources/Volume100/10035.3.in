1042666661 1853096536
1337234042 201073678
756267325 563291563
527714251 2083037062
805999857 1054566377
1454508325 2137914228
610957493 1317646306
1643831217 1402707091
922243926 1449529480
1683376137 587097020
1274795768 2041776260
2098862359 805124977
1827309532 1897252705
0 0
