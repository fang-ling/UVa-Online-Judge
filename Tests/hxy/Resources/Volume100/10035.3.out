3 carry operations.
3 carry operations.
3 carry operations.
4 carry operations.
7 carry operations.
4 carry operations.
3 carry operations.
3 carry operations.
5 carry operations.
5 carry operations.
6 carry operations.
5 carry operations.
5 carry operations.
